util/cron/docker-chapel.bash

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+#This script will compile and execute helloworld inside a docker image and check if chapel is installed correctly
+
+# Check if chapel is isntalled correctly
+which chpl
+if [ $? -ne 0 ]
+then
+      echo "which chpl failed inside chapel container"
+      exit 1
+    fi
+
+chpl --version
+if [ $? -ne 0 ]
```

Comment:
If these `if`s stay around they should be indented consistently, and in the other files as well. I think the usual way is for `if`, `then`, `else`, and `fi` to be unindented, with all the code blocks between them indented one level.

Reply:
Fixed indentation.